question.html

```diff
@@ -98,63 +98,75 @@ <h1 style="font-family: 'Bubblegum Sans', cursive;"><b>Let's ASK!</b></h1>
+					<img class="media-object" src="http://www.tiptopglobe.com/forum/images/avatars/gallery/Simpsons/simpsons_santas_little_help.gif" alt="...">
+				</a>
 			</div>
-				<hr style="margin-top:5px;margin-bottom:5px">
-
-
-				<!-- iterate replies -->
-				<span class="list-group-item" ng-repeat="reply in todosReplies | filter:{'parentID':todo.$id} : true " style="margin-left:50px; background-color: #FFEDD6" >
-					<!-- display replies -->
-					<h5 class="list-group-item-heading">
-						<span class="badge">{{-reply.order}}</span>
-						<span ng-class="{completed: todo.completed}">{{reply.desc}}</span>
-					</h5>
-					<h5 class="list-group-item-text"></h5>
-					<small style="margin-top:5px; color:#848484">
-						<em>created: {{ reply.timestamp | date:'medium' }}</em>
-					</small>
-					<!-- display buttons for reply -->
-					<div>
-						<button class="btn btn-default btn-xs glyphicon glyphicon-hand-up" ng-click="doLikeReply(reply)" ng-disabled="$storage[reply.$id]">Like</button>
-						<button class="btn btn-default btn-xs glyphicon glyphicon-hand-down" ng-click="doDislikeReply(reply)" ng-disabled="$storage[reply.$id]">Disike</button>
-						&nbsp;
-						<button ng-show="isAdmin" class="btn btn-default btn-xs glyphicon glyphicon-trash" aria-hidden="true" ng-click="removeTodo(todo)">Del</button>
-					</div>
-
-				</span>
+			<!-- Question Body -->
+			<div class="media-body">
+				<!-- Question Heading -->
+				<h4 class="media-heading">
+					<span class="label label-danger" ng-show="todo.new">New</span>
+					<span ng-class="{completed: todo.completed}">{{todo.head}}</span>
+				</h4>
+				<div style="font-family: 'Merriweather Sans', sans-serif; font-style: italic">{{todo.desc}}<br></div>
+				<small style="margin-top:5px; color:#848484">
+					<em>created: {{ todo.timestamp | date:'medium' }}</em>
+				</small>
+				<div>
+					<button class="btn btn-success btn-xs glyphicon glyphicon-thumbs-up" ng-click="doLike(todo)" ng-disabled="$storage[todo.$id]">Like<span class="badge">{{todo.like}}</span></button>
+					<button class="btn btn-danger btn-xs glyphicon glyphicon-thumbs-down" ng-click="doDislike(todo)" ng-disabled="$storage[todo.$id]">Dislike<span class="badge">{{todo.dislike}}</span></button>
+					<span class="label label-primary glyphicon glyphicon-comment">Replies<span class="badge alert-info">{{todo.replies}}</span></span>
+					<button ng-show="isAdmin" class="btn btn-default btn-xs glyphicon glyphicon-check" ng-click="toggleCompleted(todo)">Toggle Completed</button>
+					&nbsp;
+					<button ng-show="isAdmin" class="btn btn-default btn-xs glyphicon glyphicon-trash" aria-hidden="true" ng-click="removeTodo(todo)">Del</button>
+				</div>
 
-				<!-- post a reply -->
+				<div class="list-group" style="background-color: #F5F5F5;">
+				<!-- Iterate Replies -->
+				<div class="media" ng-repeat="reply in todosReplies | filter:{'parentID':todo.$id} : true " style="padding: 10px">
+					<!--Replier Avatar-->
```

Comment:
I have mentioned it before, the horizontal divider is deprecated in Bootstrap 3, and I have tried add divider using <hr> tag in html but it looks horrible, will find a better way to redesign it in the future.